tools/travis/osx-cibuild.cmake

```diff
@@ -0,0 +1,77 @@
+##
+## this script is invoked by lnx-cibuild.sh during the main "script:" section in .travis.yml
+##
+
+# define build name&co for easier identification on CDash
+set(CTEST_BUILD_NAME "$ENV{BUILD_NAME}")
+
+set(CTEST_SITE "travis-ci-build-server")
+set(CTEST_SOURCE_DIRECTORY "$ENV{SOURCE_DIRECTORY}")
+set(CTEST_BINARY_DIRECTORY "${CTEST_SOURCE_DIRECTORY}/_build")
+
+message(STATUS "CTEST_SOURCE_DIRECTORY: ${CTEST_SOURCE_DIRECTORY}")
+message(STATUS "CTEST_BINARY_DIRECTORY: ${CTEST_BINARY_DIRECTORY}")
+
+
+#/usr/local for homebrew
+set(INITIAL_CACHE
+"CMAKE_PREFIX_PATH=$ENV{SOURCE_DIRECTORY}/contrib\;/usr/local
```

Comment:
this seems to be the only difference between the osx vs lnx file.
So to reduce maintenance effort and code duplication, could this be solved via a variable in 'cibuild.sh'?